src/main/java/net/jodah/failsafe/function/Predicates.java

```diff
@@ -0,0 +1,11 @@
+package net.jodah.failsafe.function;
+
+public class Predicates {
+  public static <T, U> java.util.function.BiPredicate<T, U> convertToJavaBiPredicate(BiPredicate<T, U> failsafeBiPredicate) {
```

Comment:
It doesn't matter too much, but maybe add these as a default method called `toJavaUtil` on the interfaces themselves. This is what Guava does for `Optional`